server/swap/swap_test.go

```diff
@@ -2517,6 +2525,7 @@ func TestTxMonitored(t *testing.T) {
 	// Confirm both redeem txns up to SwapConf so they are no longer monitored.
 	matchInfo.db.makerRedeem.coin.setConfs(int64(rig.abc.SwapConf))
 	matchInfo.db.takerRedeem.coin.setConfs(int64(rig.xyz.SwapConf))
+	sendBlock(rig.abcNode) // trigger process block to remove them
```

Comment:
Note this change in behavior.  Now the match needs to be pruned from the matches map before it is no longer found by TxMonitored.  Meeting the confirmations threshold no longer makes it not monitored.